docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
+
+   **Note:** Continue scrolling down on this page to display the **Assign color** field.
+
+3. In the **Assign color** field, use the drop-down menu to select a color for your new network. The color helps you quickly identify your network from the rest of the networks in the list. Select a color and click **Submit**.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-managenet-color.png" alt="Cobalt ManageNet Color" width="300" height="200" />
+</p>
+
+#### Terms of Use
+
+The Cobalt wallet provides the **Terms of Use** webpage. This page contains the term usage, legal information, and user restrictions required to use Cobalt. 
+
+1. In the **Account** menu, click **Terms of Use**. The **Horizen - Privacy Policy** web page appears.
+
+![cobalt-termofuse](/img/docs/get-started/cobalteon-terms-policy.png)
+
+2. Read and understand the information contained in the Terms of Use and accept. 
+
+### Import Token
+
+The Cobalt wallet allows for importing ERC-20 tokens. Before importing a token, make sure that you set the network in the Cobalt wallet with the corresponding contract’s network. 
+
+To illustrate Cobalt wallet’s versatility, this example uses a contract address in the Mumbai Testnet network. The Chainlink LINK token is imported. After the token import, the balance of LINK and MATIC (the native token) are retrieved from the Mumbai Testnet network.
+
+1. In the **Assets** page, click **Import token**.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-import-matic.png" alt="Cobalt ImportToken Matic" width="300" height="200" />
+</p>
+
+2. The **Import token** page appears. Enter the **contract address** and click **Import**.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-import-token-address.png" alt="Cobalt ImportToken Address" width="300" height="200" />
+</p>
+
+
+3. The **Import token** page displays the import token details. Review the token information. Click **Import**.
+
+* **Contract Address** - The address hosting the contract on the chain that executes functions.
+* **Token Symbol** - The name of the token for coin (currency).
+* **Token Decimals** - The amount of decimals a token has (divisible from 0 to 18)
+* **Balance** - The amount of tokens owned in the selected network.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-import-proceed.png" alt="Cobalt ImportToken Proceed" width="300" height="200" />
+</p>
+
+4. Once the import transaction is complete, the Cobalt wallet displays the imported token in the Assets page.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-import-complete.png" alt="Cobalt ImportToken Complete" width="300" height="200" />
+</p>
+
+### Send ZEN
+
+The Cobalt wallet allows you to send or transfer ZEN as well as other native tokens to wallets in their respective networks. 
+
+Before sending a transaction, make sure that the appropriate network for recipient address is set in your Cobalt wallet.
+
+To send ZEN, perform the following steps:
+
+1. In the **Assets** page, click on the **Asset row**. 

```

Comment:
I'm not sure users would automatically know what **Asset row** is in this case. should we expound this or update the image to highlight `TZEN 0.51` to clearly identify what **Asset row** is?

Reply:
Added to text to clarify the meaning of Assets row by using example